WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PhaseStepping/StepConnectionConfirmationTests.cs

```diff
@@ -54,12 +55,15 @@ public async Task NotAllConfirmedStaysAsync()
 			round.Alices.Add(a3);
 			round.Alices.Add(a4);
 			round.SetPhase(Phase.ConnectionConfirmation);
-			using Arena arena = await WabiSabiFactory.CreateAndStartArenaAsync(cfg, round);
+
+			Prison prison = new();
```

Comment:
(It is useful that we can use a mock here too. Althouht certainly possible with some modifications on master too.)